www_src/pages/discover/mock.json

```diff
@@ -0,0 +1,126 @@
+[
+    {
+        "id": 1,
+        "version": "0.0.0",
```

Comment:
:+1: to maintaining a version history. Not included in v1 but something I'd like to consider in the future. Agreed to simplifying the version to just be a number that we increment.